src/Shared/CertificateGeneration/UnixCertificateManager.cs

```diff
@@ -47,12 +67,23 @@ internal override void CorrectCertificateState(X509Certificate2 candidate)
 
         protected override bool IsExportable(X509Certificate2 c) => true;
 
-        protected override void TrustCertificateCore(X509Certificate2 certificate) =>
-            throw new InvalidOperationException("Trusting the certificate is not supported on linux");
+        protected override void TrustCertificateCore(X509Certificate2 certificate, IReporter reporter, bool isInteractive)
+        {
+            using var pemFile = new PemCertificateFile(certificate);
+            foreach (var store in CertificateStores)
+            {
+                reporter.Output($"Installing into {store.StoreName}");
+                store.TryInstallCertificate(CertificateStoreKey, pemFile, reporter, isInteractive);
+                // TODO: handle failure.
```

Comment:
Is this supposed to throw?
Probably we want to try all locations, even if there is one that throws.

Reply:
The general pattern that we follow here is to capture three things:
1. Begin operation
2. Success
3. Failure

I think its ok to continue trying to setup trust in other stores even if one of them fails.

For reference, I had issues with the nssdb store being corrupted in the past when trying to setup the cert.

Reply:
The logging doesn't show up unless the user increases verbosity, right?

What should the user see when the install was partial, and when it failed completely?
Should something be printed using the `IReporter`? Should these have different exit codes?